Adds a 'Getting Started' guide to Vocabulary (#774)
creativecommons/vocabulary: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx

import { Meta } from '@storybook/addon-docs/blocks'

<Meta title="Vocabulary/Getting Started" />

# Getting Started

We hope you've grasped what `Vocabulary` is all about. Let's now dive straight into building a simple web application with it.

## Preface

> You've learnt how to install `Vocabulary` and the various ways of doing the same in the previous section. You're now _almost_ ready to build your first `CC Vocabulary` powered application!

Reviewer: ```suggestion
You've learnt multiple ways to install `Vocabulary` in the previous section. You're now _almost_ ready to build your first `CC Vocabulary` powered application!
```

Made this sentence a little simpler 👍

> Throughout this tutorial, you will learn the _"how to's"_ when it comes to using our components. It's really not that difficult, we promise!

Reviewer: ```suggestion
Throughout this tutorial, you will learn the _"how-tos"_ when it comes to using our components. It's really not that difficult, we promise!
```
